src/common/API_functions.c

```diff
@@ -222,6 +222,34 @@ int MMG5_Set_inputSolName(MMG5_pMesh mesh,MMG5_pSol sol, const char* solin) {
   return 1;
 }
 
+/**
+ * \param mesh pointer toward the mesh structure.
+ * \param fparamin name of the input solution file.
+ * \return 1 if success, 0 if fail
+ *
+ * Set the name of input solution file.
```

Comment:
Update Doxygen doc (wrong doc copy-paste ;-)). `solution` has to be replaced by `parameter`.